CHANGELOG.asciidoc

```diff
@@ -166,6 +194,10 @@ https://github.com/elastic/beats/compare/v7.6.2...v7.7.0[View commits]
 - Release vsphere module as GA. {issue}15798[15798] {pull}17119[17119]
 - Add PubSub metricset to Google Cloud Platform module. {pull}15536[15536]
 - Add dashboard for `redisenterprise` module. {pull}16752[16752]
+- Add test for documented fields check for metricsets without a http input. {issue}17315[17315] {pull}17334[17334]
+- Add final tests and move label to GA for the azure module in metricbeat. {pull}17319[17319]
+- Added documentation for running Metricbeat in Cloud Foundry. {pull}17275[17275]
+- Reference kubernetes manifests mount data directory from the host when running metricbeat as daemonset, so data persist between executions in the same node. {pull}17429[17429]
```

Comment:
```suggestion
- In the kubernetes manifests, mount the data directory from the host, so data persist between executions in the same node. {pull}17429[17429]
```

Reply:
Looking at the PR, it doesn't appear to me that this is limited to running as daemonset, so I've made this description consistent with the description under Auditbeat.